manifests/init.pp

```diff
@@ -1,37 +1,45 @@
 # @summary
 #  Init class for installing the IBM Installation Manager.
 #
-# @example Installing Installation Manager to the default location of '/opt/IBM/InstallationManager'. In this example, we've downloaded and  extracted the Installation Manager packages (installer) to '/vagrant/ibm/IM'
+# @example Installing Installation Manager to the default location of '/opt/IBM/InstallationManager'.
+#   In this example, we've downloaded and  extracted the Installation Manager packages (installer) to '/vagrant/ibm/IM'
```

Comment:
All of these comments, do they need to take a new line?

Reply:
It get's a warning if the don't

Reply:
A puppet warning is fine, the hard return may cause issues with 'puppet-strings' doc generation and looks untidy. We can submit it with the warnings, it wont cause failures. It also keeps the code uniform. This will need applied to all the comments. Sorry.